src/client/cli/cmd/unalias.cpp

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2021 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "unalias.h"
+#include "common_cli.h"
+
+#include <multipass/cli/argparser.h>
+#include <multipass/platform.h>
+
+namespace mp = multipass;
+namespace cmd = multipass::cmd;
+
+mp::ReturnCode cmd::Unalias::run(mp::ArgParser* parser)
+{
+    auto ret = parse_args(parser);
+    if (ret != ParseCode::Ok)
+    {
+        return parser->returnCodeFrom(ret);
+    }
+
+    if (aliases.remove_alias(alias_to_remove))
+    {
+        try
+        {
+            MP_PLATFORM.remove_alias_script(alias_to_remove);
+        }
+        catch (std::runtime_error&)
```

Comment:
As mentioned before, it would be good to get the error message here and print it out as well.

Reply:
`remove_alias_script` calls `unlink`. Should we inspect `ERRNO` to see the reason in case of failure? Just letting the exception propagate will not add much verbosity here...

Reply:
Ok, but what I'm saying is that `remove_alias_script()` throws a `runtime_error` with a message and that message is completely ignored here.  Why not at least combine that error message with the following error message so as not to basically duplicate part of the string.  For example, this is something like what I'm thinking:
```
catch (std::runtime_error& e)
{
    cerr << fmt::format("Warning: {} for '{}'\n", e.what(), alias_to_remove);
}
```
and perhaps change the message that `remove_alias_script()` sets to not have "error" in the wording.  Also, since `remove_alias_script` is a platform function, it may be useful to capture `errno` and add that to the error message.

Reply:
Excellent, done.